test

```diff
@@ -20,7 +20,7 @@ source ./build
 etcd_setup_gopath
 
 if [ -z "$PASSES" ]; then
-	PASSES="fmt dep compile build unit"
+	PASSES="dep compile build unit"
```

Comment:
probably keep this the same, just have `PASSES="dep compile build unit" GOARCH=amd64 ./test` in the travis file